http-gateway/web/README.md

```diff
@@ -0,0 +1,39 @@
+# PLGD Dashboard
+
+## Requirements
+
+    nodejs
+
+## Configuration
+Configuration for the client can be found in `./src/auth_config.json`.
+
+## Installation
+
+    $ npm install
+
+## Starting the development server
+
+    $ npm start
+
+Application will be hosted on `http://localhost:3000` by default. To change the default port, put `PORT=xxxx` into `package.json` script for starting the development server
+
+    cross-env PORT=3000 craco start
+
+or set `PORT` into your enviroment variables.
+
+## Building the app
+
+    $ npm build
+
+## Translation
+For extracting the messages from the UI components, run the following script:
+
+    $ npm run generate-pot
+
+This script will generate a `template.pot` file, which contains all the strings from the application ready to be translated. Upload this file to your translation tool, translate the strings and after that, export the `.po` files for all the translations and place them into `./i18n` folder.
+
+For generating language files which are used by the application, run the following script:
+
+    $ npm run generate-language-files
```

Comment:
When I upload new translation files, do I need to always run `npm run generate-language-files` or `npm build` is enough?

Reply:
Good question and good point, I will put the `npm run generate-language-files` into the `npm build` script so that you don't have to run these separately (otherwise, you would have to).

Reply:
@ondrejtomcik FYI updated the scripts.